CODEOWNERS

```diff
@@ -4,7 +4,7 @@ bigquery/   @jsimonweb
 bigtable/   @jsimonweb

```

Comment:
Shouldn't these be set the same as you did in https://github.com/googleapis/google-cloud-dotnet/pull/4947?

Reply:
@crwilcox to confirm, I think we hadn't received info from other teams about which should be included

Reply:
Yep going to add those.